app/views/account/fragments/planList.scala.html

```diff
@@ -0,0 +1,34 @@
+@import com.gu.memsub.subsv2.SubscriptionPlan
+@import views.support.Dates._
+@import views.support.Pricing._
+@import com.gu.memsub.Benefit.PaperDay
+@import org.joda.time.LocalDate.now
+
+@(plans: List[SubscriptionPlan.ContentSubscription], singularLabel: String, chosenPaperDays: List[PaperDay] = List.empty)
+
+@if(plans.length > 1) {
+    <dt class="mma-section__list--title">@{
+        singularLabel + "s"
+    }</dt>
+}
+@if(plans.length == 1) {
+    <dt class="mma-section__list--title">@singularLabel</dt>
+}
```

Comment:
the dt part could be common, and this could be something like `@if(plans.length == 1) (singularLabel) else (singularLabel + "s")`